views/documentation_word.rhtml

```diff
@@ -0,0 +1,440 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <style media="screen">
+      a {
+        color: blue;
+      }
+
+      .subtitle {
+        font-size: 19px;
+        color: rgb(110, 115, 115);
+      }
+
+      .sub-subtitle {
+        margin-top: 12px;
+        font-size: 17px;
+        color: rgb(110, 115, 115);
+      }
+
+      .title-doc {
+        line-height: 400px;
+        vertical-align: middle;
+        text-align: center;
+      }
+
+      .date-creation {
+        text-align: center;
+        font-weight: normal;
+        font-size: 20px;
+      }
+
+      .content-page {
+        margin-left: 30px;
+      }
+
+      .mt-5 {
+        margin-top: 5px;
+      }
+
+      .technology-page {
+        margin-left: 40px;
+        margin-top: 20px;
+      }
+
+      .mb-5 {
+        margin-bottom: 5px
+      }
+
+      .mt--5 {
+        margin-top: -5px;
+      }
+    </style>
+  </head>
+  <body>
+    <div class='title-page'>
+      <h1 class='title-doc'><%= Rails.application.class.parent_name %></h1>
+      <h2 class='date-creation'><%= "#{Date.today.day}th #{Date::MONTHNAMES[Date.today.month]} #{Date.today.year}" %></h2>
```

Comment:
`th` cannot be used always, for more details looks at [here](https://www.ego4u.com/en/cram-up/vocabulary/numbers/ordinal)